Readme.md

```diff
@@ -4,14 +4,18 @@
 
 ## Important Notice:
 
-You must have Node installed via `nvm` for this project to work correctly 
+You must have Node installed via `nvm` for this project to work correctly
 ([Setup Instructions](https://github.com/nvm-sh/nvm#installing-and-updating))
 
-
 ## Prerequisites
 
-- Create an access token for the GitLab API. This will be required to install our private NPM packages. The token has to have API scope.  
-Link to the GitLab settings: [Manage GitLab Access Tokens](https://gitlab.mediacube.at/-/profile/personal_access_tokens)
+- Create an access token for the Github API. This will be required to install our private NPM packages. You can follow [this](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) guide.
+
+  TLDR:
+
+  - Go to settings -> Developer settings -> Personal access tokens -> Tokens (classic)
+  - create a token with at least the scope `read:packages`
+  - `npm login --scope=@OWNER --auth-type=legacy --registry=https://npm.pkg.github.com` with your github user and the token as password
```

Comment:
```suggestion
  - `npm login --scope=@a11yphant --auth-type=legacy --registry=https://npm.pkg.github.com` with your github user and the token as password
```

We could insert the correct scope in the command here. I can already see myself trying to login for the `@OWNER` scope...